schema_registry.hh

```diff
@@ -166,6 +166,7 @@ schema_registry& local_schema_registry();
 class global_schema_ptr {
     schema_ptr _ptr;
     unsigned _cpu_of_origin;
+    schema_ptr _base_schema;
```

Comment:
nitpick: if the order here isn't important (for the sake of some serialization or something like that), then it would look neater to have the two schema_ptr instances next to each other, and _cpu_of_origin as the last attribute.

Reply:
I think it is safe to change it - no specific dependency in the order.